newscoop/src/Newscoop/ArticlesBundle/Resources/public/app/comments-ctrl.js

```diff
@@ -232,7 +232,8 @@ angular.module('editorialCommentsApp').controller('EditorialCommentsCtrl', [
 	        	$scope.textareaMessage = {};
 	        	$scope.textareaReply = {};
 	        	comment.isReplying = false;
-	        	$scope.enableTyping = false
+	        	$scope.enableTyping = false;
+                $scope.stopRefreshing = false;
```

Comment:
it's like that because some lines are tabed with tabs and some with spaces, please change file tabs to spaces everywhere.